src/osd/ReplicatedPG.cc

```diff
@@ -6215,7 +6219,12 @@ void ReplicatedPG::_copy_some(ObjectContextRef obc, CopyOpRef cop)
     // it already!
     assert(cop->cursor.is_initial());
   }
-  op.copy_get(&cop->cursor, get_copy_chunk_size(),
+
+  __le32 copyget_flags = 0;
```

Comment:
uint32_t .  the byte swabbing happens automagically when it gets assigned to the args.copy_get field.